fem/tmop_tools.cpp

```diff
@@ -266,7 +266,7 @@ void ParAdvectorCGOper::Mult(const Vector &ind, Vector &di_dt) const
 
    if (al == AssemblyLevel::PARTIAL)
    {
-      K.ParFESpace()->GetParMesh()->DeleteGeometricFactors();
+      K.ParFESpace()->GetParMesh()->NodesUpdated();
    }
```

Comment:
Ditto

> Is the `if` statement necessary? Seems simpler/clearer to just always call `NodesUpdated`, if the form is not using PA, then the geometric factors won't be requested, and no harm will be done...

Reply:
Changed in https://github.com/mfem/mfem/pull/3235/commits/8a01807a924921ede00b0566367d6bee979c3526